### PR TITLE
Add holodeck-gw recipe

### PR DESCRIPTION
**Pull request by davecwright3**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by davecwright3** [comment]
I am willing to  be listed as a maintainer

**Comment by davecwright3** [comment]
@conda-forge/help-python there seems to be ongoing issues with resolving dependencies. Could you provide any assistance?

File: recipes/holodeck-gw/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    - astropy
+    - cosmopy
+    - h5py
+    - kalepy
+    - matplotlib-base
+    - scipy
+    - pydoe
+    - psutil
+    - tqdm
+    - cython <3.0.0
```
Comment (ocefpaf) [inline recipes/holodeck-gw/meta.yaml:39]: You probably don't need cython at run-time.

```suggestion
```

File: recipes/holodeck-gw/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools >=40.6.0
+    - wheel
+    - cython <3.0.0
+    - numpy <2.0.0
```
Comment (ocefpaf) [inline recipes/holodeck-gw/meta.yaml:25]: Just a test, I believe that this packages dependencies may be outdated and an older numpy may help. The "right" fix would be updating the feedstocks of the dependencies.

```suggestion
    - numpy 1.24.4
```

**Comment by davecwright3** [comment]
@conda-forge/help-python the issues have been resolved and this is ready for review

**Comment by davecwright3** [comment]
@conda-forge/help-python this is ready for review

**Pull request closed without merging**